src/views/InProgress.vue

```diff
@@ -27,52 +27,52 @@
           </ion-label>
           <ion-icon :icon="printOutline" />
         </ion-item>
-      </div> 
-
+      </div>
       <ion-button expand="block" class="desktop-only" fill="outline" @click="packOrdersAlert">{{ $t("Pack orders") }}</ion-button>
-
-      <ion-card>
+      <ion-card v-for="(orders, index) in inProgressOrders.list.inProgress" :key="index">
         <div class="card-header">
           <div class="order-primary-info">
             <ion-label>
-              Darooty Magwood
-              <p>{{ $t("Ordered") }} 27th January 2020 9:24 PM EST</p>
+              {{ orders.doclist.docs[0].customerName }}
+              <p>{{ $t("Ordered") }} {{ $filters.formatUtcDate(orders.doclist.docs[0].orderDate, 'YYYY-MM-DDTHH:mm:ssZ', 'Do MMMM YYYY LT z') }}</p>
             </ion-label>
           </div>
 
           <div class="order-tags">
             <ion-chip outline>
               <ion-icon :icon="pricetagOutline" />
-              <ion-label>NN10584</ion-label>
+              <ion-label>{{ orders.doclist.docs[0].orderId }}</ion-label>
             </ion-chip>
           </div>
 
-          <div class="order-metadata">
+          <!-- <div class="order-metadata">
             <ion-label>
               Next Day Shipping
               <p>{{ $t("Ordered") }} 28th January 2020 2:32 PM EST</p>
             </ion-label>
-          </div>
+          </div> -->
         </div>
 
         <div class="box-type desktop-only">
           <ion-button fill="outline"><ion-icon :icon="addOutline" />{{ $t("Add Box") }}</ion-button>
           <ion-chip> Box A | Type 3</ion-chip>  
         </div>
 
-        <div class="order-item">
-          <div class="product-info">
-            <ion-item lines="none">
-              <ion-thumbnail>
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
-              <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
-              </ion-label>
-            </ion-item>
-          </div>
+        <div v-for="order in orders.doclist.docs" :key="order">
```

Comment:
We don't need an extra div for the v-for loop we can define this `v-for` condition in the div with class `order-item` and remove this extra div.

Reply:
Done Sir